source/localizable/analytics/privacy_policy.html.md.erb

```diff
@@ -15,143 +15,201 @@ title: privacy_policy
+Skroutz Analytics στα «Επίδοσης» / «Ανάλυσης» / «Στατιστικών» της ιστοσελίδας
+του συνεργαζόμενου καταστήματος. Αυτά θα ενεργοποιούνται εφόσον ο χρήστης έχει
+δώσει τη συγκατάθεσή του στο Skroutz.gr, δηλαδή ανεξαρτήτως του εάν έχει
+επιλέξει τη σχετική κατηγορία ο χρήστης (και) στο site του συνεργάτη που
+χρησιμοποιεί τη λειτουργικότητα Skroutz Analytics.
+
+Αφετέρου, προτείνεται η αναγραφή πλήρους λίστας των χρησιμοποιούμενων από το
+συνεργάτη cookies, συμπεριλαμβανομένων των cookies του Skroutz Analytics, ως
+εξής:
+
+#### **Επίδοσης / Ανάλυσης / Στατιστικών**
+
+| Όνομα             | Σκοπός                                                                                                                                                                                                                                                     | Διάρκεια                           | Τρίτο μέρος που έχει πρόσβαση |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------|-------------------------------|
+| analytics_session | Διάκριση χρηστών. Υποστήριξη της λειτουργικότητας Skroutz Analytics, κατόπιν της μετάβασης των χρηστών από τη Skroutz.gr.                                                                                                                                  | 1 μήνας                            | Skroutz                       |
+| __skr_nltcs_ss    | Διάκριση χρηστών. Υποστήριξη της λειτουργικότητας Skroutz Analytics, κατόπιν της μετάβασης των χρηστών από τη Skroutz.gr.                                                                                                                                  | 1 μήνας                            | Skroutz                       |
+| _sa_meta          | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | Μέχρι το κλείσιμο του φυλλομετρητή | Skroutz                       |
+| __skr_nltcs_mt    | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | Μέχρι το κλείσιμο του φυλλομετρητή | Skroutz                       |
+| sa_session        | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | 1 μήνας                            | Skroutz                       |
```

Comment:
There is a typo  here, it's `_sa_session`

Also the ttl of this cookie is 30 minutes, and the description should be the same as `analytics_session` and `__skr_nltcs_ss`.

🤔 I think `Skroutz Analytics -` is redundant for the `_sa_meta`, `__skr_nltcs_mt` and `_sa_session`.

Reply:
... or perhaps we should add this prefix to all the Skroutz Analytics' cookies